src/pages/RepoPage/FailedTestsTab/FailedTestsPage/FailedTestsTable/FailedTestsTable.tsx

```diff
@@ -129,11 +133,14 @@ const columns = [
       </div>
     ),
     cell: (info) => {
+      const value = (info.renderValue() ?? 0) * 100
+      const isInt = Number.isInteger(info.renderValue())
+
       return (
         <Tooltip delayDuration={0} skipDelayDuration={100}>
           <Tooltip.Root>
             <Tooltip.Trigger className="underline decoration-dotted decoration-1 underline-offset-4">
-              {info.renderValue() ? `${info.renderValue()}%` : '9%'}
+              {isInt ? `${value}%` : `${value.toFixed(2)}%`}
```

Comment:
Thoughts on swapping over to using [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat)? 

It's a really nifty built-in tool in all browsers for handling just this sort of thing, so you shouldn't have to do any ternaries etc.

Reply:
Looked into it a bit and it looks like the equivalent code would be something like

```
      const formatter = new Intl.NumberFormat('en-US', {
        style: 'percent',
        maximumFractionDigits: 2,
      }).format(info.renderValue() ?? 0)
```

It's a neat tool for sure, but I think the cognitive load from deconstructing this code is actually higher than what I had previously